hikari/traits.py

```diff
@@ -547,6 +548,9 @@ async def start(
         afk : builtins.bool
             The initial AFK state to display in the bot user presence, or
             `builtins.False` (default) to not show any.
+        check_for_updates : builtins.bool
+            Defaults to `builtins.True`. If `builtins.True`, will check for
```

Comment:
I don't think we need to enforce all implementations require checking for updates.

Probably best to revert this file.

Reply:
Will revert, will write this documentation in bot.py instead :+1: